shared/profile/index.js

```diff
@@ -4,6 +4,14 @@ import {connect} from 'react-redux'
 import Render from './render'
 import type {Props} from './render'
 import flags from '../util/feature-flags'
+import {getProfile} from '../actions/tracker'
+
+// TEMP
+const usernames = ['chromakode', 'max', 'jzila', 'mikem', 'strib', 'zanderz', 'gabrielh', 'chris',
```

Comment:
this is just to help test. when you go to the profile screen on refresh it'll be a different user

- [ ] remove before merging